CONTRIBUTING.md

```diff
@@ -105,7 +105,7 @@ To build Cats you should have
  * `console`: launch a REPL
  * `test`: run the tests
  * `unidoc`: generate the documentation
- * `fmt`: run formatting of the code
+ * `scalafmt`: run formatting of the code
```

Comment:
Noticed that `fmt` does not work for me locally, but `scalafmt` does, so I went ahead and tossed an update here, can remove if unnecessary.

Reply:
Can do!